prow/lib.sh

```diff
@@ -14,6 +14,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+function setup_gcloud_credentials() {
```

Comment:
This pulls https://github.com/maistra/istio/commit/ed5002c1499766aa9a7bee842b4045cdeb125618#diff-09361e4e1d0f5c6062a58779c9b2b6aa2a959d7364762158cde572cbfe816808 in so that prow tests will run

Reply:
was this change necessary to run locally or on prow?

Reply:
It was necessary to run on prow.